lang/java/reef-common/src/main/java/org/apache/reef/util/OSUtils.java

```diff
@@ -47,11 +47,12 @@ public static boolean isWindows() {
    * @return true, if the JVM is running on Linux. false, otherwise
    */
   public static boolean isLinux() {
-    return System.getProperty("os.name").toLowerCase().contains("linux");
+    final String os = System.getProperty("os.name").toLowerCase();
+    return os.contains("linux") || os.contains("mac");
```

Comment:
This is dangerous. Shouldn't we better add a method `isMac()`? Also, we can have `isUnix()` to indicate that the OS is either Mac OS or Linux.